core/src/main/scala/org/apache/spark/sql/eventhubs/EventHubsSource.scala

```diff
@@ -165,21 +165,27 @@ private[spark] class EventHubsSource private[eventhubs] (sqlContext: SQLContext,
             s"version from $text.")
         }
       }
-
-    metadataLog
-      .get(0)
-      .getOrElse {
-        // translate starting points within ehConf to sequence numbers
-        val seqNos = ehClient.translate(ehConf, partitionCount).map {
-          case (pId, seqNo) =>
-            (NameAndPartition(ehName, pId), seqNo)
-        }
-        val offset = EventHubsSourceOffset(seqNos)
-        metadataLog.add(0, offset)
-        logInfo(s"Initial sequence numbers: $seqNos")
-        offset
+    val defaultSeqNos = ehClient
```

Comment:
Can you please create unit tests for the scenario?

Reply:
What exactly part you need unit tests?

Reply:
What exactly unit tests should include for the scenario?

Reply:
added